doc/developer-notes.md

```diff
@@ -497,6 +497,10 @@ make cov
 # unit and functional tests.
 ```
 
+Additional LCOV options can be specified using `LCOV_OPTS`, but may be dependant
+on the version of LCOV. For example, when using LCOV `2.x`, branch coverage can be
+enabled by setting `LCOV_OPTS="--rc branch_coverage=1"`, when configuring.
+
```

Comment:
Not only "when configuring". Providing `LCOV_OPTS` variable when building the `cov` or `cov_fuzz` targets also works and improves flexibility:
```
./configure --enable-lcov CXXFLAGS="-fprofile-update=prefer-atomic"
make
make cov LCOV_OPTS="--rc branch_coverage=1 --ignore-errors mismatch"
```

Reply:
> Not only "when configuring". 

Sure. This is just one example.